dali/python/nvidia/dali/_debug_mode.py

```diff
@@ -469,48 +485,53 @@ def run(self, inputs, kwargs):
         inputs = list(inputs)
 
         # Check inputs classification as batches and extract data from DataNodeDebugs.
-        for i, (input, expected_classification) in enumerate(zip(inputs, self._inputs_classification)):
+        for i, (input,
+                expected_classification) in enumerate(zip(inputs, self._inputs_classification)):
```

Comment:
This looks weird to say the least....
```suggestion
        for i, (input, expected_classification) \
            in enumerate(zip(inputs, self._inputs_classification)):
```
or, if you want to avoid backslash:
```suggestion
        for i, (input, expected_classification) in enumerate(
            zip(inputs, self._inputs_classification)):
```

Reply:
Weird is matter of opinion and unnecessary backslashes are discouraged by Pep 8. I'm going to leave it as it is.